test/extensions/filters/http/lua/lua_filter_test.cc

```diff
@@ -1337,7 +1333,6 @@ TEST_F(LuaHttpFilterTest, HttpCallImmediateResponse) {
   Http::TestResponseHeaderMapImpl expected_headers{{":status", "403"},
                                                    {"set-cookie", "flavor=chocolate; Path=/"},
                                                    {"set-cookie", "variant=chewy; Path=/"}};
-  EXPECT_CALL(decoder_callbacks_, encodeHeaders_(HeaderMapEqualRef(&expected_headers), true));
```

Comment:
The `expected_headers` is not be verified for now. Should we add a sendLocalReply mock here or remote this local variable?

Reply:
@wbpcode  sendLocalReply is being tested in other testcases. Can I remove the local variable for this case?

Reply:
> @wbpcode sendLocalReply is being tested in other testcases. Can I remove the local variable for this case?

Seems like this case is covered by the `ImmediateResponse`, right? If is, we can remove this case directly.

Reply:
I had tried it earlier but failed with below error.
```
`Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
test/extensions/filters/http/lua/lua_filter_test.cc:1317:3: error: no member named 'gmock_sendLocalReply' in 'testing::NiceMock<Envoy::Http::MockAsyncClient>'
  EXPECT_CALL(cluster_manager_.thread_local_cluster_.async_client_, sendLocalReply(_, _, _, _, _))
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
external/com_google_googletest/googlemock/include/gmock/gmock-spec-builders.h:2028:3: note: expanded from macro 'EXPECT_CALL'
  GMOCK_ON_CALL_IMPL_(obj, InternalExpectedAt, call)
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
external/com_google_googletest/googlemock/include/gmock/gmock-spec-builders.h:2020:16: note: expanded from macro 'GMOCK_ON_CALL_IMPL_'
  ((mock_expr).gmock_##call)(::testing::internal::GetWithoutMatchers(), \
   ~~~~~~~~~~~ ^
<scratch space>:51:1: note: expanded from here
gmock_sendLocalReply`
```